### ISSUE TITLE
Accuracy issues with Pytorch's "AlexNet" pre-training model.

### ISSUE THREAD

**Issue by Lornatang** [pytorch/hub#85]
When I ran the latest version of AexNet's pre-training model to test the ImageNet validation set, I found that the accuracy of top1 and top5 increased. Here are my test details:

```
Test: [  0/391] Time  3.516 ( 3.516)    Loss 1.0981 (1.0981)    Acc@1  73.44 ( 73.44)   Acc@5  91.41 ( 91.41)
Test: [ 10/391] Time  1.555 ( 1.372)    Loss 1.8061 (1.2305)    Acc@1  61.72 ( 70.88)   Acc@5  79.69 ( 88.07)
......
Test: [380/391] Time  1.443 ( 1.605)    Loss 1.1601 (1.9751)    Acc@1  69.53 ( 55.10)   Acc@5  91.41 ( 78.20)
Test: [390/391] Time  1.158 ( 1.604)    Loss 3.5698 (1.9602)    Acc@1  23.75 ( 55.41)   Acc@5  61.25 ( 78.40)
 * Acc@1 55.412 Acc@5 78.402
```
Not in the article:

|Model structure|Top-1 error|Top-5 error|
|:-------------:|:---------:|:---------:|
|    alexnet    |   43.45   |   20.91   |

but:

|Model structure|Top-1 error|Top-5 error|
|:-------------:|:---------:|:---------:|
|    alexnet    |   44.59   |   21.60  |

In this paper, all the training details I see the `source papers` and `pytorch/examples/imagenet` the code.

Can you tell me some training tricks? **Thank you!**

**Comment by ailzhang** [comment]
cc: @fmassa do you have any idea about this issue?

**Comment by Lornatang** [comment]
> cc: @fmassa do you have any idea about this issue?

The precision of the pretrained model cannot reach the accuracy given in the document

**Comment by changyu98** [comment]
please use multi crop eval. @Lornatang
